### PR TITLE
[EigenSolversApp] simplify mkl install instructions

### PR DESCRIPTION
**Pull request by armingeiser**
some simplifications

### CODE REVIEW

File: applications/EigenSolversApplication/README.md [hunk 1]
```diff
@@ -126,23 +126,22 @@ In case you have installed [MKL](https://software.intel.com/en-us/mkl), you can
...
+For more details see https://software.intel.com/en-us/articles/installing-intel-free-libs-and-python-apt-repo.
 
 ```bash
-apt-get update -y
-apt-get upgrade -y
-apt-get install -y gnupg2 software-properties-common wget
-wget https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS-2019.PUB -P/tmp
-apt-key add /tmp/GPG-PUB-KEY-INTEL-SW-PRODUCTS-2019.PUB
-rm /tmp/GPG-PUB-KEY-INTEL-SW-PRODUCTS-2019.PUB
-echo deb https://apt.repos.intel.com/mkl all main > /etc/apt/sources.list.d/intel-mkl.list
```
Comment (armingeiser) [inline applications/EigenSolversApplication/README.md:138]: @oberbichler this command did not work for me

File: applications/EigenSolversApplication/README.md [hunk 2]
```diff
@@ -126,23 +126,22 @@ In case you have installed [MKL](https://software.intel.com/en-us/mkl), you can
...
 ## Install MKL on Ubuntu with apt
 
-Open a terminal window and run the following commands to install MKL from the official Intel repository:
+Open a terminal window and run the following commands to install MKL from the official Intel repository.
+For more details see https://software.intel.com/en-us/articles/installing-intel-free-libs-and-python-apt-repo.
 
 ```bash
-apt-get update -y
-apt-get upgrade -y
-apt-get install -y gnupg2 software-properties-common wget
```
Comment (armingeiser) [inline applications/EigenSolversApplication/README.md:134]: i think those are not necessary (except wget in the next line) but i don't think we need to add that in the instruction

File: applications/EigenSolversApplication/README.md [hunk 3]
```diff
@@ -126,23 +126,22 @@ In case you have installed [MKL](https://software.intel.com/en-us/mkl), you can
...
-wget https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS-2019.PUB -P/tmp
-apt-key add /tmp/GPG-PUB-KEY-INTEL-SW-PRODUCTS-2019.PUB
-rm /tmp/GPG-PUB-KEY-INTEL-SW-PRODUCTS-2019.PUB
-echo deb https://apt.repos.intel.com/mkl all main > /etc/apt/sources.list.d/intel-mkl.list
-add-apt-repository ppa:git-core/ppa -y
```
Comment (armingeiser) [inline applications/EigenSolversApplication/README.md:139]: also this seems not to be needed

File: applications/EigenSolversApplication/README.md (hunk 1 above)
Comment (armingeiser) [reply applications/EigenSolversApplication/README.md:138]: i replaced it with the one from the intel page

Comment (oberbichler) [reply applications/EigenSolversApplication/README.md:138]: The other one is also from the intel page

**Review by oberbichler — COMMENTED** [review]
👍 for simplification

The (old) are working when running ubuntu inside docker. All of them are needed. The instructions are also working in the CIP pool.

**Comment by oberbichler** [comment]
...or something like `source <(curl -sL https://bit.ly/2OrrKxE)`

**Comment by philbucher** [comment]
Ping @armingeiser @oberbichler what is the status of this?

**Comment by armingeiser** [comment]
From my point of view it is ready, someone would need to approve.

**Comment by armingeiser** [comment]
@oberbichler @philbucher now as the mkl from conda can also be used, i think we need to update the install instructions even more?

**Review by philbucher — APPROVED** [review]
I tried this and for me it was working!
I also tried the version that is currently in master but it didn't work (I didn't really tried hard TBH ;) )

If @oberbichler agrees I would merge this
thanks @armingeiser

**Pull request merged**